lab_1/scripts/build-client-2.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# example
+# ./build-client-2.sh
+
+sourceDir=dist
+artifactName=artifact.zip
+
+npm install
```

Comment:
f we speak about this script usage cases, there is high possibility that it will be used in CI pipeline. So it's better to use "npm ci" instead of "npm install"